packages/editor/src/components/editor-help/help-detail-navigation-screen.native.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { View, Text } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+/**
+ * WordPress dependencies
+ */
+import { BottomSheet } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+const HelpDetailNavigationScreen = ( { name } ) => {
+	const navigation = useNavigation();
+
+	const goBack = () => {
+		navigation.goBack();
+	};
+
+	return (
+		<BottomSheet.NavigationScreen name={ __( 'What is a block?' ) }>
+			<BottomSheet.NavigationHeader
+				screen={ name }
+				leftButtonOnPress={ goBack }
```

Comment:
No necessarily requesting a change, but we could simplify this if desired. Merely sharing for future reference. 
```suggestion
				leftButtonOnPress={ navigation.goBack }
```

Reply:
Nice! Thanks for that. I'll apply it in my next PR 👍